src/components/BookLists.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import Book from './Book';
+import Form from './Form';
+import '../assets/styles/booklist.css';
+
+const BookLists = () => (
+  <>
+    <ul>
+      <Book>Title</Book>
+      <Book>Author</Book>
+      <Book>
+        <button type="button">Remove</button>
+      </Book>
+    </ul>
+    <Form />
+  </>
+);
```

Comment:
- [ ] Please create a component called Book for displaying a single book (receive a title and an author as prop) and reuse it in a component that displays a list of books. Kindly implement it